### PR TITLE
EOS-27789 : RGW: cortx-rgw-integration RPM build process

### PR DESCRIPTION
**Pull request by sachitanands**
Changes include , Generation of cortx-rgw-integraion rpm using python (setup.py) tool.

RPMs generated using above script
_cortx-rgw-integration-2.0.0-1_f86435d.noarch.rpm
cortx-rgw-integration-2.0.0-1_f86435d.src.rpm 
cortx-rgw-integration-2.0.0.tar.gz_

Signed-off-by: Sachitanand Shelake <sachitanand.shelake@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      },
+      long_description=long_description,
+      zip_safe=False,
+      python_requires='>=3.6',
+      entry_points={
+          'console_scripts': [
+              'rgw_setup = cortx.rgw.setup.rgw_setup:main',
+              ]
+      },
+      data_files =[ ('%s/mini-provisioner' % RGW_INSTALL_PATH, mini_prov_files),
```
Comment (shriya-deshmukh) [inline setup.py:58]: As prvsnr invoke setup script from  (/opt/seagate/cortx/<component>/bin/<component>_setup) path,
We should keep rgw_setup script in 'bin' directory without .py extension.

Comment (sachitanands) [reply setup.py:58]: done

**Comment by sachitanands** [comment]
cortx-rgw-integration rpm after installation.

image

**Comment by sachitanands** [comment]
validated these changes locally by installing the cortx-rgw-integration rpm on vm.

**Review by rahul-tripathi-git — APPROVED** [review]
Looks good to me.

**Review by gargpalak048 — APPROVED** [review]
LGTM

**Review by shriya-deshmukh — APPROVED** [review]

**Review by shailesh-vaidya — APPROVED** [review]

**Review by selvakumaar5496 — APPROVED** [review]

**Review by cdeshmukh — APPROVED** [review]

**Pull request merged**